src/Epic/index.tsx

```diff
@@ -151,20 +164,26 @@ export const Epic: React.FC<EpicProps> = (props: EpicProps) => {
                         </p>
                     ))}
                     {/* buttonText and buttonUrl will have been checked for not undefined in canRenderEpic */}
-                    <ContributionButtonWithReminder
-                        buttonText={buttonText as string}
-                        buttonUrl={buttonUrl as string}
-                        hidePaymentIcons={hidePaymentIcons}
-                        remindMeButtonText={remindMeButtonText}
-                        remindMeConfirmationText={remindMeConfirmationText}
-                        remindMeConfirmationHeaderText={remindMeConfirmationHeaderText}
-                        trackClick={(buttonId: number) =>
-                            trackClick({
-                                internalButtonId: buttonId,
-                                ophanComponentId: ophanComponentId as string,
-                            })
-                        }
-                    ></ContributionButtonWithReminder>
+                    <div css={styles.buttonWrapperStyles}>
+                        <ContributionCtaButton
+                            buttonText={buttonText as string}
+                            buttonUrl={buttonUrl as string}
+                            hidePaymentIcons={hidePaymentIcons as string}
+                            internalButtonId={PRIMARY_BUTTON_INTERNAL_ID}
```

Comment:
Would it be better to have these internal tracking IDs be part of the relevant button component (rather than pass them in)?

Reply:
We have a ticket in Trello to sort out Braze banner/epic message tracking - maybe look at this as part of that work?

Reply:
I think that ticket is more a concern about sending the right tracking data. Here I'm talking more about internal code structure/organisation. If we're always passing in a static hardcoded value maybe it'd reduce complexity to have that value be part of the component (rather than adding a prop to parameterise it)?

Reply:
Updated as suggested